stack.yaml

```diff
@@ -34,6 +34,7 @@ extra-deps:
   commit: 8ad2378cee5ccf7937d9e08aacd5c5b7128318e8
 
 resolver: lts-19.33
+allow-newer: true
```

Comment:
Is this changed to be merged?

Reply:
Yes. Temporary due to bounds in liquid-base, which we either patch there or we live with allow-newer until my next PR, which is just waiting for this one to be merged, and which reverts this particular line.